README.md

```diff
@@ -393,6 +393,17 @@ The `all.json` file of the given hash. Other files, such as `missionRewards.json
 }
 ```
 
+## For Developers
+
+### Dependencies
+- [Node.js >= 14](https://nodejs.org/en/)
+
+### Usage
+You can run the build script with `npm run start`.
+Once built you serve the app Web Ui with `npm run serve:dev` or `npm run serve`
+
+**The npm run serve needs a $PORT in your terminal env*
```

Comment:
```suggestion
*The `npm run serve` script requires the `PORT` environment variable to be defined. `npm run serve:dev` provides this through nodemon.*
```

Reply:
I'm also a bit concerned because `npm run serve` is for deployment, not development

Reply:
Yeah the dev script solves this issue, it just doesn't in windows which would be another issure apart.

Reply:
Runs fine I'm my windows environment